core/src/com/unciv/logic/city/IConstruction.kt

```diff
@@ -18,26 +20,29 @@ open class SpecialConstruction(override var name: String, val description: Strin
     override fun shouldBeDisplayed(construction: CityConstructions): Boolean {
         return isBuildable(construction)
     }
+    open fun getProductionTooltip(cityInfo: CityInfo) : String
+            = "\r\n${(cityInfo.cityStats.currentCityStats.production / CONVERSION_RATE).roundToInt()}/${"{turn}".tr()}"
```

Comment:
you can remove the {} brackets around "turn" since it's a single phrase

Reply:
Oh, I saw the brackets in the turnOrTurns method so assumed that was needed.  Thanks for the info.